docs/utilities/data_classes.md

```diff
@@ -248,6 +250,189 @@ Verify Auth Challenge | `data_classes.cognito_user_pool_event.VerifyAuthChalleng
         do_something_with(user_attributes)
     ```
 
+#### Define Auth Challenge Example
+
+=== "app.py"
+
+    ```python
+    from aws_lambda_powertools.utilities.data_classes.cognito_user_pool_event import DefineAuthChallengeTriggerEvent
+
+    def handler(event, context):
+        _event: DefineAuthChallengeTriggerEvent = DefineAuthChallengeTriggerEvent(event)
```

Comment:
Could you please add a comment here as to why you're creating a new `_event` var?

I understand why knowing Cognito but others may not. Perhaps a class method to build what's needed or `_event.__dict__` for an easier UX

Reply:
Added `raw_event` method.